src/main/webapp/WEB-INF/views/css/header.css

```
@@ -0,0 +1,25 @@
+nav {
+    float: right;
+    margin-right: 15%;
+}
+div {
+    padding: 10px;
+    background-color: #e5ea65;
+    height: 40px;
+}
```

But usually there is a one line placed between css blocks
```suggestion
nav {
    float: right;
    margin-right: 15%;
}

div {
    padding: 10px;
    background-color: #e5ea65;
    height: 40px;
}
```